spec/integration/associations_spec.rb

```diff
@@ -24,6 +24,37 @@ class Person < ActiveFedora::Base
     end
 
   end
+  
+  describe "explicit foreign key" do
+    before do
+      class FooThing < ActiveFedora::Base
+        has_many :bars, :class_name=>'BarThing', :predicate=>ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf, :as=>:foothing
```

Comment:
Do we want to make "foothing" implicit, but overrideable? If the class was named "Page" then it seems implicit that `:as => :page`.